src/WingetCreateCLI/Properties/Resources.resx

```diff
@@ -925,4 +925,47 @@
   <data name="SyncForkWithUpstream_Message" xml:space="preserve">
     <value>Unable to create a reference to the forked repository. This can be caused when the forked repository is behind by too many commits. Sync your fork and try again.</value>
   </data>
+  <data name="DefaultInstallLocation_KeywordDescription" xml:space="preserve">
+    <value>Represents the default installed package location. Used for deeper installation detection</value>
+  </data>
+  <data name="FileSha256_KeywordDescription" xml:space="preserve">
+    <value>The optional Sha256 of the installed file</value>
+  </data>
+  <data name="Files_KeywordDescription" xml:space="preserve">
+    <value>List of installed files</value>
+  </data>
+  <data name="FileType_KeywordDescription" xml:space="preserve">
+    <value>The optional installed file type</value>
+  </data>
+  <data name="InstallationMetadata_KeywordDescription" xml:space="preserve">
+    <value>Details about the installation. Used for deeper installation detection</value>
+  </data>
+  <data name="InvalidZipFile_ErrorMessage" xml:space="preserve">
+    <value>The package file is not a valid zip archive.</value>
+  </data>
+  <data name="InvocationParameter_KeywordDescription" xml:space="preserve">

```

Comment:
>InvocationParameter_KeywordDescription [](http://example.com/codeflow?start=14&length=38)

There's also a DisplayName field in InstallationMetadata Files, not sure if this pr is using the latest schema

Reply:
Changed display name message to be more generic